posts/posts.robot

```diff
@@ -0,0 +1,75 @@
+*** Setting ***
+Resource    ./post_resources.robot
+Test Setup    Create HTTP Context     ${HOST}
+
+
+*** Test Cases ***
+
+C4 Retrieve collection of posts
+    [Documentation]   https://jobsity.testrail.net/index.php?/cases/view/4
+    [tags]    GET
+    
+    GET    /posts
```

Comment:
Create a keyword in the resource that takes in as arguments the method, endpoint, response code and returns the body. Like that you don't have to repeat stuff on every case like Logging the Response Body or validating the response code. Make sure this keyword can be used in other sections as well...